lib/test/Test/Pos/CryptoSpec.hs

```diff
@@ -36,7 +36,7 @@ import           Test.Pos.Util.QuickCheck.Property (qcIsLeft, (.=.))
 {-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
 
 spec :: Spec
-spec = withDefConfiguration $ describe "Crypto" $ do
+spec = withDefConfiguration $ \pm -> describe "Crypto" $ do
```

Comment:
Can't we use the `dummyProtocolMagic` here?

Reply:
The value `pm` is taken from the `configuration.yaml` and threaded through using `withDefConfiguration`. Again, not sure what the long term plan for that config file is.